config/database.yml

```diff
@@ -1,29 +1,23 @@
 development:
-  adapter: postgresql
-  database: <%= Settings.postgres.database %>
-  encoding: unicode
-  pool: <%= Settings.postgres.pool %>
-  timeout: 5000
-  username: <%= Settings.postgres.username %>
-  password: <%= Settings.postgres.password %>
-  host: <%= Settings.postgres.host %>
-  port: <%= Settings.postgres.port %>
-  properties:
-    sslcert: <%= Settings.postgres.sslcert %>
-    sslkey: <%= Settings.postgres.sslkey %>
-    sslmode: <%= Settings.postgres.sslmode %>
+  adapter: <%= Settings.devdb.adapter %>
+  host: <%= Settings.devdb.host %>
+  port: <%= Settings.devdb.port %>
+  database: <%= Settings.devdb.database %>
+  username: <%= Settings.devdb.username %>
+  password: <%= Settings.devdb.password %>
+  pool: <%= Settings.devdb.pool %>
+  timeout: <%= Settings.devdb.timeout %>
+  encoding: <%= Settings.devdb.encoding %>
```

Comment:
@frankiepun - I'm tempted to suggest that we make both `development` and `production` configurations use `Settings.oracle`, much like the configuration shared `Settings.postgres` between both before. This might simplify things a bit.

Reply:
Done. Both Development and Production are using the same settings - Settings.oracle.*